### DISCUSSION TITLE
Is auto-resampling supported for inference ?

### DISCUSSION THREAD

**Discussion by BenoitWang** [NVIDIA-NeMo/NeMo#6322] [Q&A] [answered]
Hi, I am using `transcribe_partial_audio` method for inference and I wonder if auto-resampling is supported or the user will always need to resample their audio before inference.

Also, is there a way to infer with pcm data instead of audio path please ?

Many thanks.

**Comment by titu1994** [comment, marked as answer]
Transcription with audio buffers is not currently supported. As to converting audio - you still need it to be mono channel audio and preferably wav file. But yes it will resample the audio to the required sample rate during audio load.

**Reply by BenoitWang** [reply]
Thanks.
